lib/ddr/models/solr_document.rb

```diff
@@ -113,7 +115,7 @@ def admin_policy
     end
 
     def has_children?
-      ActiveFedora::SolrService.class_from_solr_document(self).reflect_on_association(:children).present?
+      children.present?
```

Comment:
No, not germane.  Since this PR adds the "children" method, it was just an opportunity to simplify this code.  Can revert the change if you prefer.